CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.0)
+project(iotools
+	VERSION 1.8
```

Comment:
Do I understand correctly that CMake parses `VERSION` here and fills the respective variables as you mentioned in the commit message?

Reply:
Yes, the `VERSION` option[^1] to `project()` is automatically decomposed into related variables that I use in *config.h.in*.

It looks like I mistakenly referred to them with a `PACKAGE_` prefix in the commit message. I will fix that in the next update.

[^1]: https://cmake.org/cmake/help/latest/command/project.html#options

Reply:
Okay, I will look at all this closer when return from my vacation around mid-January.